### ISSUE TITLE
add sentence embedding pickle to .gitignore

### ISSUE THREAD

**Issue by spencerwueste** [priyansh007/AutoBots#1]
File is too large and takes forever uploading to git. Better to keep locally.

**Comment by priyansh007** [comment]
Yup lets try to store all this pickles n all in one folder called modelData and ignore it in gitignore
